web/packages/teleterm/src/ui/ThemeProvider/theme.ts

```diff
@@ -124,9 +124,15 @@ const colors = {
   },
 
   error: {
-    light: red['A200'],
-    dark: red['A700'],
-    main: red['A400'],
+    main: '#FF6257',
```

Comment:
`text.primaryInverse` of Connect's theme has to be updated, otherwise danger alert fails the contrast check. I suppose it could just match the dark theme.

![contrast check of alerts danger](https://user-images.githubusercontent.com/27113/232059261-b679de67-c07b-47aa-b543-0cb9d0a96c05.png)

Reply:
This seems to perform worse than the previous version. Did you try using the same black that the dark theme uses? AFAIK the original issue was that the Web UI uses #000000 while Connect had this almost-but-not-fully black color which causes those contrast issues.

![alert danger issues](https://user-images.githubusercontent.com/27113/232101988-44be8ed7-b48a-412a-a1c4-b778739e8ac7.png)

Reply:
This is now fixed, I had made a typo and accidentally set the `primaryInverse` to white instead of black.

![image](https://user-images.githubusercontent.com/56373201/232107407-c4c01545-ca27-4c62-965a-2002e3731c34.png)
![image](https://user-images.githubusercontent.com/56373201/232107520-d0bca70b-b6a6-47ac-81ef-d8de9d87babd.png)